yt_dlp/extractor/twitter.py

```diff
@@ -814,6 +814,14 @@ def _graphql_to_legacy(self, data, twid):
             'tweet_results', 'result'
         ), expected_type=dict, default={}, get_all=False)
 
+        # sometimes actual tweet might also be wrapped in "TweetWithVisibilityResults"
+        # https://twitter.com/TaylorLorenz/status/1616319245847265280
+        if result.get('tweet'):
+            result = result['tweet']
```

Comment:
```suggestion
            'tweet_results', 'result', ('tweet', None)
```
Add the commented URL as a test